tests/pr_test.go

```diff
@@ -71,7 +71,9 @@ func TestRunUpgradeCompleteExample(t *testing.T) {
 		BestRegionYAMLPath: regionSelectionPath,
 		ResourceGroup:      resourceGroup,
 		TerraformVars: map[string]interface{}{
-			"pg_version": "11", // Always lock to the lowest supported Postgres version
+			"pg_version": "11",                                                                                                // Always lock to the lowest supported Postgres version
+			"users":      "[{\n    name = \"testuser\",\n    password = \"password1234\" \n    type     = \"database\"\n  }]", // pragma: allowlist secret
+			"admin_pass": "password1234",                                                                                      // pragma: allowlist secret
```

Comment:
I think we need a way to hide this because this is going to spin up an instance and this will be a valid admin password for that instance. Can we randomly generate one in golang and never expose it here?

Reply:
I will explore this option